ui/components/modals/base/example.jsx

```diff
@@ -268,7 +268,7 @@ export let Footless = (props) => (
   <Backdrop>
     <Modal aria-labelledby="modal-heading-01">
       <ModalHeader>
-        <h1 id="modal-heading-01" className="slds-modal__title slds-hyphenate">
+        <h1 id="modal-heading-01" className="slds-modal__title slds-hyphenate" tabindex="-1">
```

Comment:
@andrewvasilchuk tabindex should be set to '0'. -1 will not make the heading focusable.

Reply:
In [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) is stated that 
> A negative value (usually `tabindex="-1"`) means that the element is not reachable via sequential keyboard navigation, but could be focused with JavaScript or visually by clicking with the mouse.

And in the docs

> add the appropriate `tabindex` attribute to the header to allow for programmatic focus

I think programmatic in this case means that it should be focusable but it should be excluded from the focus flow order of the modal.

Reply:
I have also found this [commit](https://github.com/salesforce-ux/design-system/commit/b60f817812968471f9b4dc006130c666b956ecc4) in which the same logic was implemented. But for  some reason the changes were reverted in this [commit](https://github.com/salesforce-ux/design-system/commit/4b65c42ae4950af96b99b8b32738eb1eb30f8109)

Reply:
Looks like that revert was a mistake. Looking at this further `tabindex=-1` should be there.